### PR TITLE
chore: bump hapi hawk

### PR DESCRIPTION
**Pull request by jackkav**
removes the following from the build output

```
npm WARN deprecated hawk@6.0.2: This module moved to @hapi/hawk. Please make sure to switch over as this distribution is no longer supported and may contain bugs and critical security issues.
npm WARN deprecated hoek@4.2.1: This version has been deprecated in accordance with the hapi support policy (hapi.im/support). Please upgrade to the latest version to get the best features, bug fixes, and security patches. If you are unable to upgrade at this time, paid support is available for older versions (hapi.im/commercial).
npm WARN deprecated boom@4.3.1: This version has been deprecated in accordance with the hapi support policy (hapi.im/support). Please upgrade to the latest version to get the best features, bug fixes, and security patches. If you are unable to upgrade at this time, paid support is available for older versions (hapi.im/commercial).
npm WARN deprecated cryptiles@3.1.4: This version has been deprecated in accordance with the hapi support policy (hapi.im/support). Please upgrade to the latest version to get the best features, bug fixes, and security patches. If you are unable to upgrade at this time, paid support is available for older versions (hapi.im/commercial).
npm WARN deprecated sntp@2.1.0: This module moved to @hapi/sntp. Please make sure to switch over as this distribution is no longer supported and may contain bugs and critical security issues.
npm WARN deprecated boom@5.2.0: This version has been deprecated in accordance with the hapi support policy (hapi.im/support). Please upgrade to the latest version to get the best features, bug fixes, and security patches. If you are unable to upgrade at this time, paid support is available for older versions (hapi.im/commercial).
```

Closes INS-1141

### CODE REVIEW

File: packages/insomnia-app/package.json [hunk 1]
```diff
@@ -77,6 +77,7 @@
     "zprint-clj"
   ],
   "dependencies": {
+    "@hapi/hawk": "^8.0.0",
```
Comment (dimitropoulos) [inline packages/insomnia-app/package.json:80]: we also need to add the types: `"@types/hapi__hawk": "^8.0.2",` to `devDependencies`

doing so uncovers a new problem Screenshot_20211103_111827

And, indeed, the types appear to be correct https://github.com/mozilla/hawk/blob/main/lib/client.js#L122.


I don't see anything in the release notes for this, but it was hard to find changes so maybe I missed it.

Comment (dimitropoulos) [reply packages/insomnia-app/package.json:80]: if you look at the 6.0.2 code https://github.com/mozilla/hawk/blob/d251edf711826e961ee6267d29a7fa50ff8cf9d0/lib/client.js#L132 it appears that they this is the same value that's now just `header`.  That suggest to me that we might be able to just do
image

but please take a deeper look.

Comment (dimitropoulos) [reply packages/insomnia-app/package.json:80]: yep:

Screenshot_20211103_112600

https://github.com/mozilla/hawk/commit/f72cdeafd178bbd893e8c041510f2ecd733c0011#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L174-R174

That's what we should do, it looks like.

Comment (dimitropoulos) [reply packages/insomnia-app/package.json:80]: [`7724e71` (#4175)](https://github.com/Kong/insomnia/pull/4175/commits/7724e719b17179599e11121c54ab7ed45451324e) resolves

**Review by dimitropoulos — APPROVED** [review]

**Comment by jackkav** [comment]
Thanks @dimitropoulos !!

**Pull request merged**